docs/openapi.yml

```diff
@@ -3451,6 +3458,22 @@ paths:
                           priceUsd: 323.92670749
                           dailyPriceChangePercent: 0.21000000
                           modifiedBlock: 3043250
+                      staking:
+                        address: tTTuKbCR2UnsEByXBp1ynBz91J2yz63h1c
+                        name: Testnet Opdex Token
+                        symbol: TODX
+                        decimals: 8
+                        sats: 100000000
+                        totalSupply: "400000000.00000000"
+                        attributes:
+                          - NonProvisional
+                          - Staking
```

Comment:
market token response would have market and liquidity pool properties

Reply:
See above comment, the market response already includes the market address so using `tokenResponse` here avoids that

Reply:
Ok, my thinking was using the MarketToken response for the market's staking token because it also includes the liquidity pool address even though we don't need the market address that comes with it. 

Our client won't look up a staking pool directly like that but others might